### PR TITLE
initial data unittests

### PR DESCRIPTION
**Pull request by RSM-git**

### CODE REVIEW

File: src/data/make_dataset.py [hunk 1]
```diff
@@ -4,12 +4,13 @@
 import torchvision
 import torch
 import os
+from typing import Tuple
```
Comment (Tolsager) [inline src/data/make_dataset.py]: from typing import *

File: tests/test_data.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+import torch
+from src.data import make_dataset
+import os
+
+
+def test_download():
+    make_dataset.download_dataset()
+    assert os.path.exists("data/raw/train.pt") and os.path.exists("data/raw/test.pt")
+
+
```
Comment (Tolsager) [inline tests/test_data.py]: add skipif data is not present

File: tests/test_data.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+def test_shape():
+    train, test = make_dataset.load_dataset()
+    assert train[0][0].shape == torch.Size([3, 32, 32])
+    assert test[0][0].shape == torch.Size([3, 32, 32])
+
+
+def test_type():
+    train, test = make_dataset.load_dataset()
+    assert train[0][0].dtype == torch.float32
+    assert test[0][0].dtype == torch.float32
```
Comment (Tolsager) [inline tests/test_data.py]: use black

File: tests/test_data.py (hunk 2 above)
Comment (RSM-git) [reply tests/test_data.py]: fixed

**Pull request merged**